src/components/richText/RichText.tsx

```diff
@@ -1,16 +1,19 @@
 "use client";
 
-import { PortableText } from "@portabletext/react";
+import { PortableText, PortableTextReactComponents } from "@portabletext/react";
+import { PortableTextObject } from "@sanity/types";
+import { ReactNode } from "react";
 
 import Text from "src/components/text/Text";
 import textStyles from "src/components/text/text.module.css";
 import { useConvertSanityImageToNextImage } from "src/utils/hooks/useConvertImage";
+import { getReactNodeTextContent } from "src/utils/reactNode";
 
 import styles from "./richText.module.css";
 
 type Children = {
```

Comment:
I think this can be deleted and replaced with PortableTextBlock

Reply:
Was not sure if the custom type was intenionally different from the Sanity one. 

Can we replace the custom `PortableTextBlock` type with the Sanity type?

```ts
import { PortableTextBlock } from 'sanity';
```

Reply:
Yes, we can 👍